Tools/PostProcessing/Visualization.ipynb

```diff
@@ -34,6 +30,28 @@
     "## Read data in the simulation frame"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Instruction\n",
+    "\n",
+    "Enter the path of the data you wish to visualize below. Then execute the cells one by one, by selecting them with your mouse and typing `Shift + Enter`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plotfile = './diags/plotfiles/plt00001'\n",
+    "field = 'Ex'\n",
+    "species = 'electron'\n",
```

Comment:
I am asking out of curiosity only.
Why not?
```suggestion
    "species = 'beam'\n",
```
As the script was doing before.

Reply:
This is just illustrative, I changed it because I tested it on an example with an electron. Maybe we could keep it this way, just to avoid re-running all the tests, what do you think?

Reply:
Yes, of course!!